l10n_it_corrispettivi/models/account.py

```diff
@@ -14,6 +14,11 @@ def _default_partner_id(self):
         if not self._context.get('default_corrispettivi', False):
             # If this is not a receipts (corrispettivi), do nothing
             return False
+        domain = [('use_corrispettivi', '=', True)]
+        partner = self.env['res.partner'].search(
+            domain, order="id desc", limit=1)
+        if partner:
+            return partner.id
```

Comment:
Mi sembra sbagliato: qui l'intento è usare di default `base.public_user` cioè l'utente anonimo, per emettere ricevute senza intestazione del cliente.
Prendere a caso il primo cliente per cui è stato impostato di emettere ricevute (`use_corrispettivi`) è un'altra cosa.

Nella issue https://github.com/OCA/l10n-italy/issues/1712 si parla di odoobot, quindi questa modifica sembra scorrelata.

Reply:
@eLBati  però in ambienti multi company se si usa il comportamento attuale di default si ottengono vari errori di permessi, da li la proposta di cambiamento

Reply:
Non lo metto in dubbio, ma la soluzione proposta mi sembra sbagliata e scorrelata